pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -463,6 +470,10 @@ func (m *cgroupManagerImpl) Create(cgroupConfig *CgroupConfig) error {
 		Resources: resources,
 	}
 
+	if cgroupConfig.ResourceParameters.PodPidsLimit != nil {
```

Comment:
I think you should check the featuregate here?

Reply:
done